tasks/release.py

```diff
@@ -1047,21 +1113,6 @@ def create_rc(ctx, major_versions="6,7", patch_version=False, upstream="origin")
             code=1,
         )
 
-    # Find milestone based on what the next final version is. If the milestone does not exist, fail.
-    milestone_name = str(new_final_version)
-
-    milestone = github.get_milestone_by_name(milestone_name)
-
-    if not milestone or not milestone.number:
-        raise Exit(
-            color_message(
-                f"""Could not find milestone {milestone_name} in the Github repository. Response: {milestone}
-Make sure that milestone is open before trying again.""",
-                "red",
-            ),
-            code=1,
-        )
-
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Why was this removed?

Reply:
OK thanks !